compiler/catala_utils/uid.ml

```diff
@@ -53,6 +53,7 @@ module Make (X : Info) (S : Style) () : Id with type info = X.info = struct
     let format ppf t =
       Format.pp_open_stag ppf (Ocolor_format.Ocolor_style_tag S.style);
       X.format ppf t.info;
+      (* Format.pp_print_int ppf t.id; (* uncomment for precise uid debug *) *)
```

Comment:
Unfortunately, we can't easily change the interface of the function because it is leveraged by many functors, in particular our `Map.Make`, etc.
An option could be a global ref of course, it really depends how often this debug is getting useful.